src/c++/vdo/base/slab-depot.c

```diff
@@ -334,7 +334,11 @@ static void launch_write(struct slab_summary_block *block)
 
 	/*
 	 * Flush before writing to ensure that the slab journal tail blocks and reference updates
-	 * covered by this summary update are stable (VDO-2332).
+	 * covered by this summary update are stable. If not stable, subsequent recovery could
+	 * encounter a slab summary update that refers to a slab journal tail block that has not
+	 * actually been written. In such cases, the slab journal referenced would be assumed to be
+	 * empty, thus causing any data within the slab which predates the existing recovery
+	 * journal entries to be lost.
 	 */
```

Comment:
Just some wording tweaks.
s/If not stable, subsequent recovery/Otherwise, a subsequent recovery/
s/would be assumed to be empty, thus causing/will be treated as empty, causing/